github-push/vendor/github.com/openfaas/openfaas-cloud/sdk/events.go

```diff
@@ -1,5 +1,10 @@
 package sdk
 
+import (
```

Comment:
events.go is probably the wrong file.

How about something else .go?

Reply:
Yeah should I create separate file for that like service.go?